packages/sample-app-fabric/windows/SampleAppFabric/SampleAppFabric.cpp

```diff
@@ -106,19 +91,17 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   // Start the react-native instance, which will create a JavaScript runtime and load the applications bundle
   host.ReloadInstance();
 
-  // Create a RootView which will present a react-native component
-  winrt::Microsoft::ReactNative::ReactViewOptions viewOptions;
-  viewOptions.ComponentName(mainComponentName);
-  auto rootView = winrt::Microsoft::ReactNative::CompositionRootView(compositor);
-  rootView.ReactViewHost(winrt::Microsoft::ReactNative::ReactCoreInjection::MakeViewHost(host, viewOptions));
+  // Create a ReactNative Island
+  auto reactNativeIsland =
+      winrt::Microsoft::ReactNative::ReactNativeIsland::Create(compositor, window, host, mainComponentName);
 
   // Update the size of the RootView when the AppWindow changes size
-  window.Changed([wkRootView = winrt::make_weak(rootView)](
+  window.Changed([wkreactNativeIsland = winrt::make_weak(reactNativeIsland)](
```

Comment:
pretty tiny naming nit:

```suggestion
  window.Changed([wkReactNativeIsland = winrt::make_weak(reactNativeIsland)](
```